ocaml/lambda/printlambda.ml

```diff
@@ -93,6 +96,7 @@ let field_kind ppf = function
   | Pgenval -> pp_print_string ppf "*"
   | Pintval -> pp_print_string ppf "int"
   | Pfloatval -> pp_print_string ppf "float"
+  | Parrayval elt_kind -> fprintf ppf "%s-array" (array_kind elt_kind)
```

Comment:
Is there a reason to put a dash here but not in the other cases ?

Reply:
It just seemed clearer (also it doesn't match up with the `floatarray` type to avoid confusion, although actually I think the representations might be the same).